### DISCUSSION TITLE
copilot could not connect to sever.

### DISCUSSION THREAD

**Discussion by YBaCuO** [community/community#14839] [Copilot Conversations]
my copilot could not connect to sever. 
![image](https://user-images.githubusercontent.com/94754810/163110468-516bcbf1-a142-43eb-8f97-de47b8bfc343.png)
how could i solve this problem?

**Comment by lucalves** [comment]
Hi @YBaCuO, how are you? I hope you're well.

If you are using VSCode, try:

1. Disable the extension
2. Reload VSCode
3. Re-enable copilot extension

If the error persists, try logging out and back in from your GitHub account in VSCode:

<img width="506" alt="VSCode Screenshot" src="https://user-images.githubusercontent.com/17712401/163112152-e3942a0e-05b7-4dea-a17e-6bda1b7849da.png">

**Reply by YBaCuO** [reply]
sorry, i've tried both approaches.  neither of them worked
